src/eval.rs

```diff
@@ -246,7 +246,7 @@ impl MainThreadState {
                 let exit_code = this.read_target_isize(&ret_place)?;
                 // Need to call this ourselves since we are not going to return to the scheduler
                 // loop, and we want the main thread TLS to not show up as memory leaks.
-                this.terminate_active_thread()?;
+                this.terminate_active_thread(true)?;
```

Comment:
```suggestion
                this.terminate_active_thread(/* leak_tls */ true)?;
```
anonymous bools are an anti-pattern; either use an enum or at least add comments that indicate their meaning